[2022-07-19 06:47:19,191] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v02.insert_into_table scheduled__2022-07-18T00:00:00+00:00 [queued]>
[2022-07-19 06:47:19,218] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v02.insert_into_table scheduled__2022-07-18T00:00:00+00:00 [queued]>
[2022-07-19 06:47:19,218] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 06:47:19,218] {taskinstance.py:1244} INFO - Starting attempt 1 of 6
[2022-07-19 06:47:19,218] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 06:47:19,265] {taskinstance.py:1264} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-07-18 00:00:00+00:00
[2022-07-19 06:47:19,277] {standard_task_runner.py:52} INFO - Started process 544 to run task
[2022-07-19 06:47:19,287] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v02', 'insert_into_table', 'scheduled__2022-07-18T00:00:00+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/example_postgres.py', '--cfg-path', '/tmp/tmpcmgwtmnx', '--error-file', '/tmp/tmpejy7rxp0']
[2022-07-19 06:47:19,288] {standard_task_runner.py:77} INFO - Job 51: Subtask insert_into_table
[2022-07-19 06:47:19,385] {logging_mixin.py:109} INFO - Running <TaskInstance: dag_with_postgres_operator_v02.insert_into_table scheduled__2022-07-18T00:00:00+00:00 [running]> on host 0741836b2d3b
[2022-07-19 06:47:19,564] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kimuksung2
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v02
AIRFLOW_CTX_TASK_ID=insert_into_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-18T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-18T00:00:00+00:00
[2022-07-19 06:47:19,593] {base.py:79} INFO - Using connection to: id: postgres_localhost. Host: host.docker.internal, Port: 5432, Schema: test, Login: ***, Password: ***, extra: {}
[2022-07-19 06:47:19,687] {dbapi.py:225} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-07-18', 'dag_with_postgres_operator_v02')
        , parameters: None
[2022-07-19 06:47:19,704] {dbapi.py:233} INFO - Rows affected: 1
[2022-07-19 06:47:19,768] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v02, task_id=insert_into_table, execution_date=20220718T000000, start_date=20220719T064719, end_date=20220719T064719
[2022-07-19 06:47:19,917] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-19 06:47:19,994] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
